### PR TITLE
Test corrections and README update

### PR DESCRIPTION
**Pull request by beccaelenzil**
Changes in this PR
- correction: `is` changed to `==` in Wave 01 tests
- correction: last two tests in Wave 05 call correct function
- Exception raised for incomplete tests
- README updated to not spell out the number of tests per wave

Still to look at:
- Bringing in the edge case tests that were added for C16 and accidentally dropped for C17

### CODE REVIEW

**Review by kaidamasaki — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -322,9 +306,7 @@ In `party.py`, there should be a function named `get_friends_unique_watched`. Th
 
 ### Wave 4
 
-1. There are four tests about a `get_available_recs` function
-
-Create a function named `get_available_recs`
+1. Create a function named `get_available_recs`. This function should...
 
 - takes one parameter: `user_data`
```
Comment (kelsey-steven-ada) [inline README.md:311]: I don't know if this is worth fixing with this PR, but I noticed there's some word agreement mismatching here and on the function descriptions below that the function descriptions above don't have.

```suggestion
- take one parameter: `user_data`
```

**Review by kelsey-steven-ada — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (beccaelenzil) [reply README.md:311]: Thanks! I'll take a closer look.

Comment (beccaelenzil) [reply README.md]: changes takes to take
